src/server/api/routes/users/v2/nfts/display-state.js

```diff
@@ -23,9 +20,9 @@ const NFT_DISPLAY_STATE = {
 
 const router = Router();
 
-router.get('/:wallet/nfts/display-state', async (req, res, next) => {
+router.get('/nfts/display-state', async (req, res, next) => {
```

Comment:
We do want to keep `/:wallet/` in public endpoints though? Since these are really under some kind of wallet without implicit session